app/main/views/index.py

```diff
@@ -19,6 +19,17 @@
 from app.main import main
 from app.utils import assess_contact_type, bytes_to_pretty_file_size
 
+FILE_EXTENSION_TO_PRETTY_FILE_TYPE = {
+    'csv': 'CSV',
+    'doc': 'Microsoft Word document',
+    'docx': 'Microsoft Word document',
+    'odt': 'text file',
```

Comment:
I think this might be better described as an OpenOffice Document file rather than a plain text file? It's definitely not the same as eg a .txt file.

Reply:
@saimaghafoor do you have thoughts on this please?

Some background on odt files:
https://docs.fileformat.com/word-processing/odt
https://fileinfo.com/extension/odt

I have no idea how familiar users might be with the terminology 'OpenOffice Document files' but equally it is a bit more serious than a plain text file as can contain things like images and more formatting.

Reply:
We actually don't show the file name because they are meaningless (we store files as a random collection of letters and numbers currently). We might work to change that in the future but for now that's why we just describe the file type rather than showing a nicer 'name'.

Just for your context, we're working off the content that Karl wrote up here: https://docs.google.com/document/d/1zBDNl7qvqXYANL_hke0Mr2oNtXbx4aVhKA7WaU33GLY/edit?disco=AAAAaal3eSU

So we already have pretty much settled on the format `Download this ((filetype)) (((file size))) to your device` - I'm just challenging the description of 'text file' for ODT files, as I think that's overly simplistic/untrue. ODTs are broadly the same kind of file format as DOC/DOCX (microsoft word).

Reply:
Ah got it. Sorry I'm not up to speed with all this yet. So an ODT file has more stuff in it than just text. Technically, is it called anything else, or fit into another category other than a text file? If we do have an alternative and decide to change here, we'll need to change all instances it's mentioned across notify too.

Reply:
I don't _think_ we describe the file anywhere else in Notify, just here, which is why we're trying to work out what's the best way to describe it :)

Reply:
Is an odt commonly referred to as a text file? Is the alternative suggested 'open document' widely used instead? Has this come up before and maybe some research done? Sorry I know there's lots of questions. I don't want to agree on something that I don't have much user research/data on :)

Reply:
.odt is to OpenDocument as .doc/.docx is to Microsoft Word, basically (for some more info maybe see https://support.microsoft.com/en-us/office/differences-between-the-opendocument-text-odt-format-and-the-word-docx-format-d9d51a92-56d1-4794-8b68-5efb57aebfdc).

Maybe if it's too complicated/complex/we aren't sure, it's fine to stick with just 'text file'. But it feels a little basic perhaps - if we're calling out Microsoft Word I think we could be justified in calling out OpenDocument too.

Reply:
I think if we're distinguishing between .odt, .txt and .rtf in brackets, we should be OK to continue with it being called a text file. 
We could always keep an eye on any feedback we get from users :)

Reply:
Thanks, I will keep it as `text file` for now then unless we learn something to change our minds.